### ISSUE TITLE
klikercentar.rs: breakage

### ISSUE THREAD

**Issue by dusanpudar** [uBlockOrigin/uAssets#25265]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://klikercentar.rs/
```

### Description

Dear uBlockOrigin Team,

last night we were hacked, where malware was introduced and these two files were infected:

```
https://klikercentar.rs/sgf.exe
https://klikercentar.rs/vrgeh.exe
```

We deleted the malware, but the site is still blacklisted, so please remove us if we are blacklisted.

Thank you, 
Dusan 

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Capture](https://github.com/user-attachments/assets/857ccc66-cf95-4927-a5c5-94a8aa51835c)

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml

uBlock Origin: 1.59.0
Chromium: 128
filterset (summary):
 network: 151034
 cosmetic: 48200
 scriptlet: 22278
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 39941-142, 19m Δ
  ublock-badware: 11052-6, 19m Δ
  ublock-privacy: 1126-2, 19m Δ
  ublock-unbreak: 2471-0, 19m Δ
  ublock-quick-fixes: 96-0, 19m Δ
  easylist: 85897-573, 19m Δ
  easyprivacy: 53047-780, 19m Δ
  urlhaus-1: 25995-0, 1h.19m
  plowe-0: 3542-0, 19m
filterset (user): [empty]
trustedset:
 removed:
  chrome-extension-scheme
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1537 ms
 maxAssetCacheWait: 430 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 113
 network:
  klikercentar.rs: 108
  googletagmanager.com: 1
  hs-analytics.net: 1
  hs-banner.com: 1
  hsadspixel.net: 1
  hsforms.com: 1

```
</details>

**Comment by stephenhawk8054** [comment]
Please report to https://gitlab.com/malware-filter/urlhaus-filter

**Comment by stephenhawk8054** [comment]
Create a new issue ticket here: https://gitlab.com/malware-filter/urlhaus-filter/-/issues

**Comment by dusanpudar** [comment]
Thank you!

**Comment by dusanpudar** [comment]
When I'm trying to login on gitlab, it said that I need to join or post some projects, I don't want that, just for the issue..

**Comment by stephenhawk8054** [comment]
The domain is blocked by that list. I don't know other ways to report to that list maintainer except reporting to that repo.

**Comment by dusanpudar** [comment]
Can you report for me, or take report from this here?

**Comment by mapx-** [comment]
Maybe a PR here:
https://github.com/curbengh/urlhaus-filter/pulls

**Comment by mapx-** [comment]
Or log in gitlab using gh credentials

**Comment by dusanpudar** [comment]
> Or log in gitlab using gh credentials

I was loged with gh..

**Comment by dusanpudar** [comment]
You don't want to help? I don't understand how to open the ticket, It is very confusing..

**Comment by JobcenterTycoon** [comment]
The site is already unblocked on my end.

**Comment by stephenhawk8054** [comment]
Yeah, I don't see the domain in https://malware-filter.gitlab.io/urlhaus-filter/urlhaus-filter-ag-online.txt any more
